src/core/base/render_base.cpp

```diff
@@ -433,7 +433,11 @@ void AbstractRender_t::lazyLoad(StdPicture &target)
 
     bool shrink2x = false;
 
-    if(g_videoSettings.scaleDownAllTextures || GraphicsHelps::validateFor2xScaleDown(sourceImage, StdPictureGetOrigPath(target)))
+    if(g_videoSettings.scaleDownAllTextures 
+#if !defined(VITA)
```

Comment:
Ha! Why don’t you just say “#if 0” here? ;)

Reply:
I dont think this one is displaying properly on GitHub. But, something tells me this was a clause I threw in while debugging and didn't mean to leave in. Oops. The intention was to skip texture scale down on Vita to avoid wasting CPU cycles on that. The Vita has plenty of VRAM for the scaled textures to not impact. I wish I had better profiling tools too though!

Reply:
No, sorry, I was joking a bit here. This wastes CPU cycles on all the platforms, and really should be cached to disk in some way. (Really, the result of the full process of “load png/gif, apply mask, scale if possible” should probably be performed on first start and cached to disk.)

Reply:
Caching support is a really interesting idea, so, can be attempted to implement separately. Looks like it wants to just save as a sort of BMP or other uncompressed image format that keeps RGBA support (I will need to store also hashes of original pics to avoid missing of the case when they got changed). Will make an issue later to explain all ideas at one post.

Reply:
The reason I put this in:

When setting `scale-down-textures` to false in the ini, I found that textures were still being sized down. I can take it out if you guys want.

Reply:
The `scale-down-textures` setting only changes the enable/disable of the 2pix validation check, and it avoids non-2pix sprites being scale-downed to avoid distortions, and when enabled, this check is ignored and textures are always scaling down to save more RAM and don't care about distortions